mongodb/src/test/java/org/apache/calcite/adapter/mongodb/MongoAdapterTest.java

```diff
@@ -284,37 +284,6 @@ private CalciteAssert.AssertThat assertModel(URL url) {
             "CITY=LAWTON; LONGITUDE=null; LATITUDE=null; POP=45542; STATE=OK; ID=73505");
   }
 
-  @Disabled("broken; [CALCITE-2115] is logged to fix it")
```

Comment:
I don't understand why we directly deleted the test cases that couldn't run. A better approach should be to make them run as we want.

Reply:
@NobiGo I found sales_for_1997 is in foodmart-data-json-0.4.jar, so I popular it, then test can run correctly.